templates/index.twig

```diff
@@ -6,6 +6,7 @@
     <div piwik-notification notification-id="ga-importer-help" context="info" type="transient" noclear="1" notification-title="{{ 'GoogleAnalyticsImporter_SettingUp'|translate }}">
         {{ 'GoogleAnalyticsImporter_ImporterHelp1'|translate }}
         {{ 'GoogleAnalyticsImporter_ImporterHelp2'|translate('<a href="https://matomo.org/docs/google-analytics-importer/" target="_blank" rel="noreferrer noopener">', '</a>')|raw }}
+        {{ 'GoogleAnalyticsImporter_ImporterHelp3'|translate('<br><br><strong>', '</strong>')|raw }}
```

Comment:
I think it's probably better to not put the <br> in the translate and just wrap the fields in <p>'s or have the <br>'s outside the translation

Reply:
@justinvelluppillai Sorry I didn't get you here

Reply:
I meant like this, but no need to worry about it now it's merged:
```suggestion
        <br><br>{{ 'GoogleAnalyticsImporter_ImporterHelp3'|translate('<strong>', '</strong>')|raw }}
```